circuit_knitting_toolbox/entanglement_forging/cholesky_decomposition.py

```diff
@@ -34,10 +34,14 @@
 from .entanglement_forging_ansatz import EntanglementForgingAnsatz
 from .entanglement_forging_operator import EntanglementForgingOperator
 
+from qiskit_nature import settings
+
+settings.use_pauli_sum_op = False
+
```

Comment:
Since it may violate a user's expectations to modify global state like this, it deserves some comments here of why it's necessary and also deserves mention in the release notes.  (At least it is a temporary thing, and will have no effect on Qiskit Nature versions 0.7 and higher.)

Reply:
Also, it might be better to move this one level up, to `entanglement_forging/__init__.py`, for increased visibility.